package.json

```diff
@@ -37,12 +37,16 @@
     "eslint-plugin-import": "2.17.2",
     "eslint-plugin-prettier": "3.0.1",
     "eslint-plugin-react": "7.13.0",
-    "husky": "2.3.0",
+    "husky": "^3.0.8",
```

Comment:
Could you add a comment here to explain why this was updated?

Reply:
oo didn't see that, I put it back the  olf version

Reply:
done

Reply:
no, it doesn't hurt  ,  works fine